larq_zoo/dorefanet.py

```diff
@@ -0,0 +1,183 @@
+import tensorflow as tf
+import larq as lq
+from larq_zoo import utils
+from zookeeper import registry, HParams
+
+
+@registry.register_model
+def dorefa_net(hparams, input_shape, num_classes, input_tensor=None, include_top=True):
+    def conv_block(x, features, kernel_size, strides=1, pool=False, padding="same"):
+        x = lq.layers.QuantConv2D(
+            features,
+            kernel_size=kernel_size,
+            strides=strides,
+            padding="same",
+            input_quantizer=hparams.input_quantizer,
+            kernel_quantizer=hparams.kernel_quantizer,
+            kernel_constraint=None,
+            use_bias=False,
+        )(x)
+        x = tf.keras.layers.BatchNormalization(scale=False, momentum=0.9, epsilon=1e-4)(
+            x
+        )
+        if pool:
+            x = tf.keras.layers.MaxPool2D(pool_size=3, strides=2, padding=padding)(x)
+        return x
+
+    def dense_block(x, units):
```

Comment:
This reminds me of a block in a densenet. What do you think about 
```suggestion
    def fully_connected_block(x, units):
```